src/initramfs/cmd/init/pkg/mount/mount.go

```diff
@@ -152,7 +152,7 @@ func Mount(s string) error {
 
 // Unmount unmounts the ROOT and DATA block devices.
 func Unmount() error {
-	for _, disk := range []string{constants.RootPartitionLabel, constants.DataPartitionLabel} {
+	for _, disk := range []string{constants.RootPartitionLabel, constants.DataPartitionLabel, constants.BootPartitionLabel} {
```

Comment:
I missed this in our conversation, but I was seeing `EBUSY` when trying to unmount. The fix was to unmount anything in `/` first (i.e. `/var` and `/boot`). So the slice should be:

```go
[]string{constants.BootPartitionLabel, constants.RootPartitionLabel, constants.DataPartitionLabel}
```